components/eamxx/docs/user/eamxx_cases.md

```diff
@@ -0,0 +1,5 @@
+Basics of EAMxx cases
+=====================================
+
+This section explains how to create a case which uses EAMxx as the atmosphere model,
+as well as what are the currently supported compsets and grids for EAMxx.
```

Comment:
I foresee this page consisting of not many lines: just list the currently supported compsets/grids, and maybe specify the list of currently available IC files and map files (for IC and/or I/O).

Reply:
My hope is that we can automagically list the supported compsets and grids rather than typing them out (so they don't go out of date). Listing IC files and map files would be nice, but would be hard to automate because there are so many for non-EAMxx uses.

Reply:
Yes, that'd be nice. But perhaps, while we figure out an automation tool, we can add some hard-coded documentation here...

Reply:
Sounds good, but let's not try too hard to make hard-coded documentation we'll hopefully replace. We could just choose our favorite few compsets and the ne1024pg2 grid.